src/api/ssl/__tests__/Prehled-dokumentu.test.ts

```diff
@@ -0,0 +1,45 @@
+import { Ginis } from '../../../index'
+
+jest.setTimeout(20000)
+
+describe('Prehled-dokumentu', () => {
+  let ginis: Ginis
+  beforeAll(() => {
+    console.log(
+      'Loading GINIS credentials from .env - make sure you have correct local configuration.'
+    )
+    ginis = new Ginis({
+      urls: {
+        ssl:
+          process.env['GINIS_SSL_HOST'] ??
```

Comment:
Out of the scope, but better solution here is to have setup env variables file in jest setup https://stackoverflow.com/a/58953365

Reply:
Since the tests are running against a real service, it's probably better to have it configured with actual env variables.